tests/adapter/UploadedFilesPsr7Test.php

```
@@ -140,6 +140,50 @@ public function testReturnMultipleUploadedFilesWithDifferentStructures(): void
         );
     }
 
+    public function testReturnUploadedFileForFailedUploadWithoutAccessingStream(): void
+    {
+        $uploadedFileFactory = HelperFactory::createUploadedFileFactory();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii2\extensions\psrbridge\tests\support\HelperFactory' in method 'testReturnUploadedFileForFailedUploadWithoutAccessingStream'. (undefined)

(StaticAccess)